ideascube/db_router.py

```diff
@@ -1,30 +1,44 @@
+
+_databaseToUse = {
+    ('search', 'search') : 'transient'
+}
+
 class DatabaseRouter(object):
     """
     A Database router.
     Some models do not need (and must not be backuped).
-    Those models will (must) have a _dbname class attribute set to 'transiant'.
+    So we route them to the transient base.
     This attribute will be use as database name.
     """
     def db_for_read(self, model, **hints):
         """
-        If the model as a _dbname set, use it as db for read
+        Route the table'read to the right base or the default one (None)
```

Comment:
Well it make sense to me. Hopefully because I wrote it. :)
If you think your version is more explicit, I don't see any reason to not change this sentence.